portfolio/src/main/java/com/google/sps/servlets/CreateNewComment.java

```diff
@@ -31,17 +33,32 @@
  */
 @WebServlet("/create-new-comment")
 public class CreateNewComment extends HttpServlet {  
-  
-  @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String username = request.getParameter("username"); 
-    String commentContent = request.getParameter("comment"); 
 
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html"); // set response type
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      String userEmail = userService.getCurrentUser().getEmail();
+      response.getWriter().println("Hello " + userEmail + "! Now that you're logged in, feel free to leave a comment.");   
+    } else {
+      response.getWriter().println("Please log in to leave a comment.");   
+    }
```

Comment:
Is this get method necessary? How is it different from the check-login-status servlet?

Reply:
This method returns a message prompting the user to log in/to leave a comment, and CheckLoginStatus just changes the text content of the button to "Log in" or "Log out".

Reply:
Why not just have check-login-status return true/false?  Usually you would want to handle the actual text in the client.  Specific UX components like text are not the job of the server.